src/app/admin/modals/user-settings-modal/user-settings-modal.component.html

```diff
@@ -0,0 +1,112 @@
+<div class="user-settings-modal form-horizontal">
+  <div class="modal-header">
+    <h3>{{ isNew ? "Create User" : "Edit User"}}</h3>
+  </div>
+  <div class="modal-body">
+    <div class="form-group" *ngIf="!isNew">
+      <label class="col-sm-3 control-label">Username</label>
+      <div class="col-sm-7">
+        <p class="form-control-static">{{ user.username }}</p>
+      </div>
+    </div>
+    <div class="form-group" *ngIf="isNew">
+      <label class="col-sm-3 control-label" ng-required="isNew">Username</label>
+      <div class="col-sm-7">
+        <input [(ngModel)]="user.username" type="input" class="form-control" placeholder="1744070" />
+      </div>
+    </div>
+    <div class="form-group" ng-required="isNew">
+      <label class="col-sm-3 control-label">First Name</label>
+      <div class="col-sm-7">
+        <input [(ngModel)]="user.first_name" type="input" class="form-control"
```

Comment:
There is an error when loading this modal:

<details>
  <summary>ERROR TypeError: Cannot read properties of undefined (reading 'first_name')</summary>

at UserSettingsModalComponent_Template (user-settings-modal.component.html:21:16)

at executeTemplate (core.mjs:9603:1)

at refreshView (core.mjs:9469:1)

at refreshComponent (core.mjs:10640:1)

at refreshChildComponents (core.mjs:9265:1)

at refreshView (core.mjs:9519:1)

at refreshEmbeddedViews (core.mjs:10594:1)

at refreshView (core.mjs:9493:1)

at refreshComponent (core.mjs:10640:1)

at refreshChildComponents (core.mjs:9265:1)
</details>